### ISSUE TITLE
two profiles gives error about missing stack attribute

### ISSUE THREAD

**Issue by BrannonKing** [vpelletier/pprofile#32]
When trying to run two profiles in one function, I always get this error on the 2nd one:
```
  File "/.../lib/python3.7/site-packages/pprofile.py", line 923, in _real_local_trace
    stack, callee_dict = self.stack
AttributeError: stack
```

**Comment by BrannonKing** [comment]
Actually, this was my error. I was trying to call dump_stats() twice on the same object. A cleaner error would be helpful but not necessary.

**Comment by vpelletier** [comment]
Thanks for the report.

Deleting `self.stack` when tracer is disabled is inconsistent with other uses of this attribute. It should now be consistent again.

This said, I'm not sure why calling `dump_stats` multiple times would cause this error: this error comes from one of the two tracing functions, which should normally not be active at the time stats are extracted. Maybe it was still referenced by another thread, which only be given a time slice when a second `dump_stats` was called in another thread ?

Anyway, it is a good point that any already-enabled tracing function would fail once tracing has been disabled, and I'm not sure why I did not get more errors like this. So I added two `try`..`except` blocks to handle this error.
